restapi/stories.go

```diff
@@ -86,7 +86,7 @@ func (api api) ReturnStoriesByCountries(w http.ResponseWriter, r *http.Request)
 	return rest.JSONStatusOK(stories)
 }
 
-func randomCoords(coordinate float64) float64 {
+func RandomCoords(coordinate float64) float64 {
```

Comment:
What do you think about moving this out into a helper instead of exposing a public member from the current file.